Classes/OldModel/Event.php

```diff
@@ -389,27 +389,11 @@ public function getPlacesWithCountry(): array
             return [];
         }
 
-        $countries = [];
-
-        $countryData = \Tx_Oelib_Db::selectMultiple(
-            'country',
-            'tx_seminars_sites LEFT JOIN ' .
-            'tx_seminars_seminars_place_mm ON tx_seminars_sites' .
-            '.uid = tx_seminars_seminars_place_mm.uid_foreign' .
-            ' LEFT JOIN tx_seminars_seminars ON ' .
-            'tx_seminars_seminars_place_mm.uid_local = ' .
-            'tx_seminars_seminars.uid',
-            'tx_seminars_seminars.uid = ' . $this->getUid() .
-            ' AND tx_seminars_sites.country <> ""' .
-            \Tx_Oelib_Db::enableFields('tx_seminars_sites'),
-            'country'
-        );
-
-        foreach ($countryData as $country) {
-            $countries[] = $country['country'];
-        }
+        $countryData = array_filter($this->getPlacesAsArray(), static function (array $place): bool {
+            return $place['country'] !== '';
+        });
 
-        return $countries;
+        return array_column($countryData, 'country');
```

Comment:
If you switch the `array_column` and the `array_filter`, we can reduce the anonymous function to `$country !==''`.